NoNicotineAPI/Controllers/TherapistsController.cs

```diff
@@ -30,12 +33,19 @@ public async Task<IActionResult> CreateTherapist(CreateTherapistCommand request)
         }
 
         [HttpGet]
-        [Route("{id}")]
-        public async Task<IActionResult> GetTherapist(string id)
+        [Authorize(Roles = "therapist")]
+        public async Task<IActionResult> GetTherapist()
         {
+            if (HttpContext.User.Identity is not ClaimsIdentity identity)
```

Comment:
You have a different implementation here than the one you used for patient.

Reply:
made it the same. fixed